hazelcast/src/main/java/com/hazelcast/config/MemberDomConfigProcessor.java

```diff
@@ -1160,7 +1156,8 @@ private void handleMulticast(Node node, boolean advancedNetworkConfig) {
             String value = getTextContent(att).trim();
             if ("enabled".equals(lowerCaseInternal(att.getNodeName()))) {
                 multicastConfig.setEnabled(getBooleanValue(value));
-            } else if ("loopbackmodeenabled".equals(lowerCaseInternal(att.getNodeName()))) {
+            } else if ("loopbackmodeenabled".equals(lowerCaseInternal(att.getNodeName()))
+                    || "loopback-mode-enabled".equals(lowerCaseInternal(att.getNodeName()))) {
```

Comment:
Camel case (`loobackModeEnabled`) is against the naming convention, so with YAML `loopback-mode-enabled` is also supported.